Kubernetes-CLI.adoc

```diff
@@ -0,0 +1,66 @@
+= Kubernetes CLI
+
+The Infinispan CLI currently has two modes: command-line and interactive. 
+To improve the user experience when using the Infinispan Operator we can enhance the CLI to add Kubernetes capabilities.
+
+== Native
+
+Using Quarkus we can build the CLI as a native executable for the three most common platforms: Linux x86_64, OS X, Windows.
+
+== Kubernetes mode
+
+Installing the executable as `kubectl-infinispan` allows invocation as a `kubectl/oc` plugin, e.g.:
+
+`kubectl infinispan ...`
+
+Using the Fabric8 Kubernetes Client we can automatically detect the existence of a Kubernetes configuration (default to `~/.kube/config`) and automatically connect to the Kubernetes cluster.
```

Comment:
If `~/.kube/config` is only the default, is there be a parameter or environment variable that can make the CLI use a different configuration?

Reply:
I believe it should work out of the box with `KUBECONFIG`